tools/team-builder/build_teams.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+run(){
+	
+	local OPTIND
+
+  #default values
+	output="fleet_osquery_packages"
+  header=false
+	flags+="--disable-open-folder"
+
+  #Read flags
+	while getopts s:hc:n:p:u:f:d:o:x flag
+	 do
+		case "${flag}" in
+			s) #path to csv file containing team names. Must end with newline char.
+				source=($OPTARG);;
+			h) #indicates whether there is a header included in the file. default false;
+				header=true;;
```

Comment:
I wonder if we should just remove all the actual CSV parsing and just go with the simple line-delimited text? That would allow the whole thing to be simplified. If someone has a CSV they can easily enough use Excel or Sheets to get the line-delimited text that your basic option allows.